draft-jholland-mops-taxonomy-md

```diff
@@ -0,0 +1,316 @@
+
+##Caching Systems
+
+TBD: pros, cons, tradeoffs of caching designs at different locations within
+the network?
+
+Peak vs. average provisioning, and effects on peering point congestion
+under peak load?
+
+Provisioning issues for caching systems?
+
+##Predictable Usage Profiles
+
+TBD: insert charts showing historical relative data usage patterns with
+error bars by time of day in consumer networks?
+
+Cross-ref vs. video quality by time of day in practice for some case
+study?  Not sure if there's a good way to capture a generalized insight
+here, but it seems worth making the point that demand projections can
+be used to help with e.g. power consumption with routing architectures
+that provide for modular scalability.
+
+##Unpredictable Usage Profiles
+
+Although TCP/IP has been used with a number of widely used applications that have symmetric bandwidth requirements (similar bandwidth requirements in each direction between endpoints), many widely-used Internet applications operate in client-server roles, with asymmetric bandwidth requirements. A common example might be an HTTP GET operation, where a client sends a relatively small HTTP GET request for a resource to an HTTP server, and often receives a significantly larger response carrying the requested resource. When HTTP is commonly used to stream movie-length video, the ratio between response size and request size can become quite large. 
+
+For this reason, operators may pay more attention to downstream bandwidth utilization when planning and managing capacity. In addition, operators have been able to deploy access networks for end users using underlying technologies that are inherently asymetric, favoring downstream bandwidth (e.g. ADSL, cellular technologies, most IEEE 802.11 variants), assuming that users will need less upstream bandwidth than downstream bandwidth. This strategy usually works, except when it does not, because application bandwidth usage patterns have changed. 
+
+One example of this type of change was when peer-to-peer file sharing applications gained popularity in the early 2000s. To take one well-documented case ({{RFC5594}}), the Bittorrent application created "swarms" of hosts, uploading and downloading files to each other, rather than communicating with a server. Bittorrent favored peers who uploaded as much as they downloaded, so that new Bittorrent users had an incentive to significantly increase their upstream bandwidth utilization. 
+
+One example of this type of change was when peer-to-peer file sharing applications gained popularity in the early 2000s. To take one well-documented case ({{RFC5594}}), the Bittorrent application created "swarms" of hosts, uploading and downloading files to each other, rather than communicating with a server. Bittorrent favored peers who uploaded as much as they downloaded, so that new Bittorrent users had an incentive to significantly increase their upstream bandwidth utilization. 
```

Comment:
This line appears twice, now that I'm looking it seems I missed that in the earlier pr.